pkg/cmd/attestation/verification/sigstore.go

```diff
@@ -65,13 +68,57 @@ func (v *LiveSigstoreVerifier) chooseVerifier(b *bundle.ProtobufBundle) (*verify
 	}
 	issuer := leafCert.Issuer.Organization[0]
 
-	// if user provided a custom trusted root file path, use the custom verifier
-	if v.config.CustomTrustedRoot != "" {
-		customVerifier, err := newCustomVerifier(v.config.CustomTrustedRoot)
+	if v.config.TrustedRoot != "" {
+		customTrustRoots, err := os.ReadFile(v.config.TrustedRoot)
 		if err != nil {
-			return nil, "", fmt.Errorf("failed to create custom verifier: %v", err)
+			return nil, "", fmt.Errorf("unable to read file %s: %v", v.config.TrustedRoot, err)
+		}
+
+		scanner := bufio.NewScanner(bytes.NewReader(customTrustRoots))
+		for scanner.Scan() {
+			// Load each trusted root
+			trustedRoot, err := root.NewTrustedRootFromJSON(scanner.Bytes())
+			if err != nil {
+				return nil, "", fmt.Errorf("failed to create custom verifier: %v", err)
+			}
+
+			// Compare bundle leafCert issuer with trusted root cert authority
+			certAuthorities := trustedRoot.FulcioCertificateAuthorities()
+			for _, certAuthority := range certAuthorities {
+				certAuthRoot := certAuthority.Root
+
+				if len(certAuthRoot.Issuer.Organization) == 0 {
+					continue
+				}
+
+				if certAuthRoot.Issuer.Organization[0] == issuer {
+					// We have our trusted root, but now we need to determine the policy
+					if issuer == PublicGoodIssuerOrg {
+						if v.config.NoPublicGood {
```

Comment:
What should we do if someone tries to do `gh at verify --trusted-root trusted_root.jsonl --no-public-good`? Alternately, we could make `--trusted-root` and `--no-public-good` mutually exclusive.

Reply:
I think we can solve this in two ways (i.e where to put the decision):
1. Let trusted root export chose what trusted roots to export
2. Rely on `--no-public-good` during verify

My gut feeling is to have option 2, as it's more in line with "online verification", i,.e have what you have implemented already.